README.md

```diff
@@ -1399,62 +1515,23 @@ Theoretically, Skate could automatically detect this and extend `path` by defaul
 
 
 
-## Polyfills
-
-As you may know, the only way to polyfill Mutation Observers is to use the deprecated DOM 3 Mutation Events. They were deprecated because if you insert 5k elements at once, you then trigger 5k handlers at once. Mutation Observers will batch that into a single callback.
-
-Skate requires that you BYO your own [Mutation Observers](https://developer.mozilla.org/en/docs/Web/API/MutationObserver) implementation. There are several out there:
-
-- https://github.com/webcomponents/webcomponentsjs (requires WeakMap polyfill)
-- https://github.com/megawac/MutationObserver.js
-- https://github.com/bitovi/mutationobserver
-
-
-
 ## Preventing FOUC
 
-An element may not be initialised right away. To prevent FOUC, you can add the `unresolved` attribute to any web component element and then use that attribute to hide the element in your stylesheets.
-
-```html
-<style>
-  [unresolved] {
-    opacity: 0;
-  }
-</style>
-<my-element unresolved></my-element>
-```
-
-The `unresolved` attribute will be removed after the `created()` callback is called and before the `attached()` callback is called.
-
-Additionally, after removing the `unresolved` attribute, Skate will add the `resolved` attribute. This allows you to transition your styles:
+An element may not be initialised right away. In order to prevent jank or FOUC, you can use the `defined` attribute to style your components accordingly.
 
 ```css
-[resolved] {
+my-element:not([defined]) {
+  opacity: 0;
+}
+
+my-element[defined] {
   opacity: 1;
   transition: opacity .3s ease;
 }
 ```
 
 
 
-## Ignoring Elements
```

Comment:
I don't want people to use this. I believe we're the only ones using it and its behaviour is inconsistent as it doesn't work in native.